### DISCUSSION TITLE
Tooltip creates overflow of body

### DISCUSSION THREAD

**Discussion by Emiliorth** [primefaces/community#1957] [PrimeNG] [answered]
In PrimeNG 17.16.1 when tooltip code is added at the end of <body>, for a less than second it's code takes some height, making body have overflow if your site is set to have height of 100vh
In previous versions of PrimeNG, like 15xx there was no such a problem
![20240516_123415_chrome](https://github.com/primefaces/community/assets/46076240/bfbfdb0c-93f9-4b49-a27d-09fdbbcc9cfb)

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi, in the latest PrimeNG version (17.17.0), this issue has been fixed. Could you please try again with the updated version?

Example (v17.17.0) = https://stackblitz.com/edit/yqjz5u-c2d5dk?file=src%2Fapp%2Ftooltip-position-demo.html

**Reply by Emiliorth** [reply]
Yes it works now, thank you :)
